### ISSUE TITLE
hot key bugs.

### ISSUE THREAD

**Issue by zkdpower** [stenzek/duckstation#2507]
Under latest 0.1 4643 Windows Version

IF i unchecked the "Enable Fullscreen UI" in General Setting, then start any game, during gaming, press Esc key, nothing happens, where is "Power off Confirm" prompt for exit? 

We Can see Both "Open Quick Menu" and "Power Off System"  default hot key is ESC.  This CONFLICT should be Solve ASAP.

**Comment by stenzek** [comment]
This is an issue with your settings. Don't bind two things to the same hotkey.

Neither power off nor quick menu is bound by default in Qt. If you launch nogui the first time, the quick menu will be bound to escape. But only if you haven't launched Qt first.

**Comment by zkdpower** [comment]
i mean i clear every hotkey. then
A. (1)"Enable Fullscreen UI" checked,
    (2)"Restore Default"
    (3)"Open Quick Menu" and "Power Off System" auto restore to ESC both.
B. (1)"Enable Fullscreen UI" unchecked,  
    (2)"Restore Default" 
    (3)"Open Quick Menu" auto restore to empty, "Power Off System" auto restore to ESC.

AND I CAN'T FIND "RESTORE DEFAULTS" FUNCTION IN NOGUI. CAN'T TEST NOGUI.

**Comment by zkdpower** [comment]
i delete every directory of duckstation（ include C:\xxx\Documents\… ）and reinstall， then, i clear all hotkey in setting. then, i did following two tries.

A.
(1)"Enable Fullscreen UI" checked,
(2)"Restore Default"
(3)"Open Quick Menu" and "Power Off System" auto restore to ESC both.
B.
(1)"Enable Fullscreen UI" unchecked,
(2)"Restore Default"
(3)"Open Quick Menu" auto restore to empty, "Power Off System" auto restore to ESC.

YOU SAY "Neither power off nor quick menu is bound by default in Qt." BUT WHY I FEEL THEY BOTH BIND TO ESC BY DEFAULT.

AND I CAN'T FIND "RESTORE DEFAULTS" FUNCTION IN NOGUI. CAN'T TEST NOGUI.





------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor ***@***.***&gt;; 
发送时间: 2021年8月14日(星期六) 中午12:00
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [stenzek/duckstation] hot key bugs. (#2507)





 
This is an issue with your settings. Don't bind two things to the same hotkey.
 
Neither power off nor quick menu is bound by default in Qt. If you launch nogui the first time, the quick menu will be bound to escape. But only if you haven't launched Qt first.
 
&mdash;
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by bslenul** [comment]
The caps and the double post don't seem necessary... but yeah:

* "Power Off" is bound to ESC by default, "Quick Menu" isn't bound to anything.
* If "Enable Fullscreen UI" is enabled however, clicking "Restore Defaults" will not restore this option to its default (disabled) and it will bind both "Quick Menu" and "Power Off" to ESC.
* Having ESC mapped to both "Quick Menu" and "Power Off" will show the quick menu if "Enable Fullscreen UI" is enabled, but it won't do anything if it's disabled.
* "Restore Defaults" doesn't unbind "Quick Menu" even if "Enable Fullscreen UI" is disabled, and it also doesn't unbind any default empty hotkeys.

Tested on Windows 10 only, latest dev build (929aa2f when typing this) using Qt. I'm not saying any of those are bugs, just doing a recap of what I've noticed while messing with OP's issue :o
